planet/scripts/v1.py

```diff
@@ -470,6 +475,8 @@ def get_mosaic_list_for_subscription(subscription_id):
         feed_image_conf = feed_info.get(type_)
 
         if feed_image_conf['type'] != 'mosaic':
+            # Unsure about the following lines. The exception is not
+            # raised.
             msg_format = 'The {} for this feed is not a mosaic type.'
             click.ClickException(msg_format.format(type_))
             continue
```

Comment:
I guess I should take care of this, too. Nothing reported, but there is clearly a bug in here.